_security/configuration/configuration.md

```diff
@@ -239,7 +239,7 @@ In this case, the header states that the message was signed using HMAC-SHA256.
 
 #### Payload
 
-The payload of a JSON web token contains the so-called [JWT Claims](https://self-issued.info/docs/draft-ietf-oauth-json-web-token.html#RegisteredClaimName). A claim can be any piece of information about the user that the application that created the token has verified.
+The payload of a JSON Web Token contains the so-called [JWT Claims](https://self-issued.info/docs/draft-ietf-oauth-json-web-token.html#RegisteredClaimName). A claim can be any piece of information about the user that the application that created the token has verified.
 
 The specification defines a set of standard claims with reserved names ("registered claims"). These include, for example, the token issuer, the expiration date, or the creation date.
```

Comment:
When you say 'these include...' do you need to say, 'for example'? Could you just say, 'The standard claims include the token issuer, the expiration date...' or are they examples of reserved names?

Reply:
I didn't pen these beauties. But they do need fixing. Thanks.

Reply:
These are just examples of the many registered claims that exist. I have rewritten in the hopes of achieving greater clarity.